kotlin/open-mobile-map-package/src/main/java/com/esri/arcgisruntime/sample/openmobilemappackage/MainActivity.kt

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2020 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.openmobilemappackage
+
+import android.os.Bundle
+import android.util.Log
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.esri.arcgisruntime.loadable.LoadStatus
+import com.esri.arcgisruntime.mapping.MobileMapPackage
+import java.io.File
+import kotlinx.android.synthetic.main.activity_main.*
+
+
+class MainActivity : AppCompatActivity() {
+
+  val TAG: String = MainActivity::class.java.simpleName
+
+  private lateinit var mapPackage: MobileMapPackage
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
+
+    val mobileMapFilePath = createMobileMapPackageFilePath()
```

Comment:
this could be as suggested, and then can remove the createMobileMapPackageFilePath method
```suggestion
    val mobileMapFilePath = getExternalFilesDir(null)?.path + getString(R.string.yellowstone_mmpk)
```